### PR TITLE
Create folder folder, to contain the project sources. Creation The Bu…

### PR DESCRIPTION
**Pull request by Deobap73**
…ttonStartCreating component ,hat renders an '<h6>' with the text Create your digital image here - No credit card required.

The Link class has been imported from react-router-dom. This class contains an ellipse group which will be used to create a link to the createpost route. The code will show a button that links to the create post page The button will have the text Start creating for free and when clicked, it will show a message that says Create your digital image here  No credit card required. The AIImageCreator component is a function that returns an HTML element. It has a Header, which contains the text Create amazing photos with the power of AI The main container div has two spans: one for Create amazing and another for AI. There s also a button to start creating images.The code is a React component that renders an image creator.The component is rendered with a Header, a div with the class maincontainer-parent and a div with the class maincontainer. The maincontainer-parent has a h2 tag with the text Create amazing photos and the text photos inside of it. The maincontainer has a p tag with the text image maker is a set of magical AI tools. Generate original images to scale, modify photos, expand images beyond the original, or create custom Ai models. There is also a ButtonStartCreating in this div.

### CODE REVIEW

File: client/src/components/AIImageCreator.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
+@import "../styles/media-queries";
+@import "../styles/variables";
```
Comment (oleksandra-github) [inline client/src/components/AIImageCreator.scss:2]: We need to change this part of code

Comment (oleksandra-github) [inline client/src/components/AIImageCreator.scss:2]: Text

**Review by oleksandra-github — CHANGES_REQUESTED** [review]
changes should be done untill 12.00

**Comment by Deobap73** [comment]
Done

**Review by oleksandra-github — APPROVED** [review]
Good job

**Pull request merged**